modules/prometheus-exporter_oracle/conf/04-blockingsessions.yaml

```diff
@@ -0,0 +1,18 @@
+module: oracle
+name: "Blocking(s) session(s)"
+id: blocking_sessions
+transformation: true
+aggregation: ".mean(by=['job'])"
+
+signals:
+  signal:
+    metric: oracledb_blocking_sessions_count
+    rollup: latest
+
+rules:
+  critical:
+    threshold: 0
+    comparator: ">"
+    lasting_duration: "5m"
+    description: "is detected"
```

Comment:
I think you can let the default description here.
if someone wants to increase the threshold this would be better.